### DISCUSSION TITLE
Enable GitHub's form schema for PULL_REQUEST_TEMPLATE

### DISCUSSION THREAD

**Discussion by tisonkun** [community/community#38424] [Pull Requests]
AS IS.

Modifying markdown syntax PULL_REQUEST_TEMPLATE is as clumsy as ISSUE_TEMPLATE before [GitHub's form schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema) support. If GitHub supports form schema for ISSUE_TEMPLATE, why not PULL_REQUEST_TEMPLATE?

**Comment by scarf005** [comment]
it's also tracked in #11058 and #31664 .

**Reply by tisonkun** [reply]
Thank you! This is exactly https://github.com/orgs/community/discussions/31664.
